ui/components/namespace-grant.js

```diff
@@ -0,0 +1,172 @@
+import useSWR, { useSWRConfig } from 'swr'
```

Comment:
definitely will need to refactor this with grant / resource-grant component - https://github.com/infrahq/infra/issues/2060

Reply:
I partially addressed that in this PR, merging the component for namespace and cluster grants